src/content/1.7/contribute/contribution-guidelines/project-modules.md

```diff
@@ -43,6 +43,35 @@ They can easily be applied by using the `header-stamp` binary from [PrestaShop m
 
 A list of the project contributors must be stored inside the module. Usually it's a file named CONTRIBUTORS.md . It must be up-to-date when a new version is released.
 
+## About compatibility
+
+### Semantic Versioning and BC breaks
+
+PrestaShop and its modules follows [SemVer](https://semver.org/). This means that contributions should strive not to introduce breaking compatibility changes. Therefore, each Pull Request must not introduce such changes, unless aiming to release a new major version.
+
+It is possible to introduce a change that is not backward compatible into the code for a minor or patch version, but it must be for a good reason.
+
+Valid reasons include, but not only:
+- fixing a security issue
+- fixing a major issue that cannot be fixed in a backward compatible manner
+- introducing a major new feature of high value
+
```

Comment:
@PrestaShop/prestashop-core-developers Question : Changing the property `ps_versions_compliancy` is a BC or not ?

Reply:
Should not be a BC breaks as the Core won't install an incompatible module :thinking: